src/renderer/screens/exchange/Swap2/Form/FormSelectors/FormInputs.js

```diff
@@ -27,64 +29,49 @@ function SwapButton() {
   );
 }
 
-export type toAccountType =
-  | {
-      // User already has an account to accept target currency
-      targetAccountExists: true,
-      data: {
-        account: Account | TokenAccount,
-        parentAccount: ?Account,
-        currency: ?(TokenCurrency | CryptoCurrency),
-      },
-    }
-  | {
-      // User doesn't have an account to accept target currency
-      targetAccountExists: false,
-      data: {
-        account: null,
-        parentAccount: null,
-        currency: ?(TokenCurrency | CryptoCurrency),
-      },
-    }
-  // No target currency selected
-  | null;
+export type toAccountType = {
```

Comment:
types should be Capitalized

Reply:
Nice catch. This type is no longer used, I forgot to remove it so I pushed a new **signed** review commit :)